icons/folder-cursor_dark.svg

```diff
@@ -0,0 +1,8 @@
+<svg version="1.1" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
+    <path d="m13.844 7.536-1.288-1.072A2 2 0 0 0 11.276 6H4a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h24a2 2 0 0 0 2-2V10a2 2 0 0 0-2-2H15.124a2 2 0 0 1-1.28-.464" fill="#e0e0e0"/>
+    <g fill="#424242" fill-rule="evenodd">
+        <path d="m23 28.333 7-11.667v8.1667z"/>
+        <path d="m16 16.667 7-4.6667 7 4.6667z"/>
+        <path d="m16 16.667 7 3.5v8.1667l-7-3.5z"/>
+    </g>
+</svg>
```

Comment:
Only the light version should have a prefix (`_light`)

Like this:

<img width="135" height="49" alt="image" src="https://github.com/user-attachments/assets/871fe0e4-4ea3-44bb-8ce5-43f9e3e784a5" />